libraries/standard/http/include/http_client.h

```diff
@@ -9,8 +9,9 @@
  * @brief Maximum size, in bytes, of headers allowed from the server.
  *
  * If the total size in bytes of the headers sent from this server exceeds this
- * configuration, then the status code #HTTP_SECURITY_ALERT_HEADERS is
- * returned from #HTTPClient_Send.
+ * configuration, then the status code
+ * #HTTP_SECURITY_ALERT_RESPONSE_HEADERS_SIZE_LIMIT_EXCEEDED is returned from
```

Comment:
What is the rationale for treating this outcome as a "Security Alert"? The user could possible set a lower than required configuration value in the macro...

Reply:
We spoke about this in the during the team design meeting. MCUs know what servers they are targeting and therefore know how many headers they are going to expect. Therefore a server that violates that is a bad server.